vagrant/Vagrantfile

```diff
@@ -146,29 +175,38 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       nodes: ["master1", "node1", "node2"],
     }
 
+    if deployment_type == 'origin'
+      ansible_groups[:'OSEv3:vars'][:openshift_release] ='latest'
+    end
+    print("deployment_type = %s" % (deployment_type))
+
     ansible_host_vars = {
       master1:  {
+        ansible_ssh_common_args: '"-o StrictHostKeyChecking=no"',
```

Comment:
Okay, trying to go back to syncing `.` instead of `..`, I get:

```
[vagrant@admin1 ~]$ cd /home/vagrant/sync && PYTHONUNBUFFERED=1 ANSIBLE_FORCE_COLOR=true ansible-playbook --limit="OSEv3:localhost" --inventory-file=/tmp/vagrant-ansible/inventory -v /home/vagrant/sync/install.yaml
Using /home/vagrant/sync/ansible.cfg as config file

ERROR! the role 'rhsm-subscription' was not found in /home/vagrant/sync/roles:/etc/ansible/roles:/home/vagrant/sync

The error appears to have been in '/home/vagrant/sync/install.yaml': line 18, column 5, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  roles:
  - role: rhsm-subscription
    ^ here

[vagrant@admin1 sync]$ 
[vagrant@admin1 sync]$ ls
ansible.cfg  install.retry  install.yaml  README.md  roles  tasks  Vagrantfile
[vagrant@admin1 sync]$ ls -al roles/
total 16
drwxrwxr-x. 1 vagrant vagrant  49 Nov  8 21:53 .
drwxrwxr-x. 1 vagrant vagrant 144 Nov 21 16:23 ..
lrwxrwxrwx. 1 vagrant vagrant  67 Nov  8 21:53 rhsm-repos -> ../../reference-architecture/aws-ansible/playbooks/roles/rhsm-repos
lrwxrwxrwx. 1 vagrant vagrant  74 Nov  8 21:53 rhsm-subscription -> ../../reference-architecture/aws-ansible/playbooks/roles/rhsm-subscription
[vagrant@admin1 sync]$ 
```

Did this work before?